scripts/fix_embargo_approved_logs.py

```diff
@@ -0,0 +1,47 @@
+"""
+Update existing "embargo_approved_no_user" logs to link to registered project instead
+of the registration.
+"""
+import logging
+import sys
+
+from modularodm import Q
+
+from framework.transactions.context import TokuTransaction
+
+from website.models import Node, NodeLog
+from website.app import init_app
+from scripts import utils as script_utils
+
+logger = logging.getLogger(__name__)
+
+
+def main():
+    targets = get_targets()
+    fix_embargo_approved_logs(targets)
+
+
+def get_targets():
+    return NodeLog.find(Q('action', 'eq', NodeLog.EMBARGO_APPROVED) & Q('params.user', 'eq', None))
+
+
+def fix_embargo_approved_logs(targets):
+    for log in targets:
+        node_id = log.params['node']
+        node = Node.load(node_id)
+        if node.is_registration:
+            log.params['node'] = node.registered_from_id
+            log.params['registration'] = node._id
+            log.save()
+            logging.debug('Updated node param of log {}'.format(log._id))
```

Comment:
As mentioned in my previous comment, use `logger.info`. Also, it would be useful to log out what params was before and after the migration.

Also, prefer to log *before* making a db change (e.g. "Updating param of log {} ..."). That way, if the script crashes on the `.save()` we can diagnose which record failed.